client/src/main/java/org/apache/rocketmq/client/consumer/store/RocksDBOffsetSerializeWrapper.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.rocketmq.client.consumer.store;
```

Comment:
Putting this class in the client module is a bit strange

Reply:
Because I found OffsetSerializeWrapper under the client moudle, so I put RocksDBOffsetSerializeWrapper in this directory as well.

Reply:
OffsetSerializeWrapper is used in client module, RocksDBOffsetSerializeWrapper is used in broker module instead of client module, so I think this is not appropriate.

Reply:
OK